crates/ika-core/src/dwallet_mpc/mpc_session.rs

```diff
@@ -86,6 +87,10 @@ pub(super) struct DWalletMPCSession {
     // TODO (#539): Simplify struct to only contain session related data - remove this field.
     weighted_threshold_access_structure: WeightedThresholdAccessStructure,
     pub(crate) mpc_event_data: Option<MPCEventData>,
+    pub(crate) received_more_messages_since_last_retry: bool,
```

Comment:
change this var to `received_more_messages_since_last_advance`
also do it, and explain why do we need it.